src/kirby/components/chart/chart.component.spec.ts

```diff
@@ -65,13 +65,13 @@ describe('ChartComponent', () => {
   });
 
   it('should have dataLabels enabled as default', () => {
-    expect(component.options.plotOptions.pie.dataLabels.enabled).toBe(true);
+    expect(component.options.plotOptions.series.dataLabels).toEqual({ enabled: true });
```

Comment:
Why not as before?
```suggestion
    expect(component.options.plotOptions.series.dataLabels.enabled).toBeTrue();
```

Reply:
Interface change in Highcharts 8

Reply:
Changed to pie.dataLabels as before, but still need the typecast